test/langtools/tools/javac/processing/model/util/elements/overrides/S.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2023, 2025, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Is the beginning copyright year 2023 correct?

Reply:
Yes, it is. The patch attached to the JBS issue is dated 2023. That patch is used in this PR.

Reply:
Thanks for the verification.